MultiOracle.md

```diff
@@ -449,6 +449,104 @@ of `t` oracles, or else they can be set as constant parameters across all groupi
 
 TODO: Decide how this is done and communicated. 
 
+### Multi Oracles with varying number of digits
+
+The above described algorithms all assume that all oracles use the same number of digits to represent numerical event outcomes.
+However that might not always be the case and dealing with oracles with varying number of digits require further care.
+
+When a contract is created with multiple oracles with varying number of digits, the contract should only define payouts for outcomes up to the maximum value that can be attested by the oracle with the minimum number of digits (thereafter referred as `min_nb_digits`).
+
+#### Equal outcomes
+
+When no difference is allowed between the oracles, one simply needs to "pre-pad" the outcome prefixes of oracles with larger number of digits with `0`s.
+For example, for the prefix `1010`, if an oracle uses `min_nb_digits + 2` digits, the prefix `001010` should be used.
```

Comment:
Mmmm so `00` is the prefix of the prefix somehow. Because `1010` is already a prefix for `1010xxxx...`, or at least that's what I was trying to convey. I updated let me know if it's clearer.